packages/server/codegen.yml

```diff
@@ -8,6 +8,7 @@ generates:
       - 'typescript'
       - 'typescript-resolvers'
     config:
+      enumsAsConst: true
```

Comment:
TS native Enums kinda suck cause values typed as those enums don't accept the same exact values, just because they're not using the enum. So this won't work:

```ts
const Foo {
 A = 'a'
}

// ERROR: Even tho the value fits the type, I used a literal string 'a', not Foo.A, and this breaks it
const test: Foo = 'a' 
```

We tend to do mapping between GQL enums and our own domain types in a lot of places, so I decided to adjust codegen config to not use native TS enums and generate const objects instead

```ts
const Foo = {
  A: 'a'
} as const

type Foo = (typeof Foo)[keyof typeof Foo]

const test: Foo = 'a' // works!
```

Reply:
def makes it a lot better, i'm still not behind importing these constants into the business logic parts of the app.